frontend/src/components/LinkButton.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import '../styles/LinkButton.css'
+
+class LinkButton extends React.Component {
```

Comment:
This component looks good, but I don't think we will ever need state here. Maybe we could turn it into a stateless component?